src/reducers/ModulesReducer.js

```diff
@@ -49,7 +49,7 @@ const initialState = () => {
     usingPayments,
     usingDashboard,
     usingDispensary,
-    usingVaccines,
+    usingVaccines: true,
```

Comment:
Hehe, is there a way to check if dev here? Either way don't want to hold up PR/Dev 😅